R/textstat_lexdiv.R

```diff
@@ -388,6 +389,7 @@ compute_lexdiv_tokens_stats <- function(x, measure = c("MATTR", "MSTTR"),
 
     # reorder output as originally supplied
     result <- result[, c("document", measure), drop = FALSE]
+    result[is.na(result)] <- NA
```

Comment:
Doesn't `is.na(result)` trigger conversion of data.frame to matrix? If so, it can be more efficient.

Reply:
no, it just replaces the values, does not drop the data.frame class.